src/DataWrangling/ETOPO/ETOPO.jl

```diff
@@ -0,0 +1,81 @@
+module ETOPO
+
+using Scratch
+export ETOPOBathy
+
+import ClimaOcean.DataWrangling:
+    metadata_filename,
+    default_download_directory,
+    metadata_path,
+    Metadata,
+    Metadatum,
+    all_dates,
+    download_dataset
+
+using Oceananigans.DistributedComputations: @root
+
+download_ETOPO_cache::String = ""
+function __init__()
+    global download_ETOPO_cache = @get_scratch!("ETOPO")
+end
+
+ETOPO_bathymetry_variable_names = Dict(
+    :temperature => "temperature",
+    :salinity    => "salinity"
+)
+
+struct ETOPOBathy end
```

Comment:
perhaps `ETOPOBathymetry` is a better name? Bathy means nothing to me.
Or perhaps `ETOPO2022`? That's the dataset we are downloading, right?

Reply:
For sure, just the name for the dataset. For other cases we don't add qualifiers like "bathymetry" or "ocean state" so I suggest just the name of the dataset and no more